go/mcap/writer.go

```diff
@@ -434,7 +433,8 @@ func (w *Writer) flushActiveChunk() error {
 	crc := w.compressedWriter.CRC()
 	compressedlen := w.compressed.Len()
 	uncompressedlen := w.compressedWriter.Size()
-	msglen := 8 + 8 + 8 + 4 + 4 + len(w.opts.Compression) + 8 + compressedlen
+	topFieldsLen := 8 + 8 + 8 + 4 + 4 + len(w.opts.Compression) + 8
```

Comment:
What is "topFields"? That's not an mcap spec term I am familiar with. Might a comment be helpful here - especially as it relates to all these magic numbers being added together.